testbench/rest_server.py

```diff
@@ -0,0 +1,897 @@
+# Copyright 2020 Google LLC
```

Comment:
Same question here

Reply:
Fixed.  I thought `git` was going to treat that as a move, but I never argue with the robots. Fixed, and reverted the change in `emulator.py`.